p2p/simulations/test.go

```diff
@@ -0,0 +1,59 @@
+package simulations
+
+import (
+	"github.com/ethereum/go-ethereum/p2p"
+	"github.com/ethereum/go-ethereum/p2p/enode"
+	"github.com/ethereum/go-ethereum/p2p/enr"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// NoopService is the service that does not do anything
+// but implements node.Service interface.
+type NoopService struct {
+	useChannel bool
+	C          map[enode.ID]chan struct{}
```

Comment:
Actually, it's used in `network_test.go`. And the idea is that this should be available outside of the package, which means it would need to be exported. But you mean that as long as that's not the case it should be private?